.github/ISSUE_TEMPLATE/add-new-platform.md

```diff
@@ -0,0 +1,44 @@
+# Adding a new supported platform(N)
```

Comment:
`(N)`?

Maybe "Implementing a new supported platform" and `implement-new-platform.md`?

Reply:
I like it, done.